webapp-django/crashstats/manage/tests/test_views.py

```diff
@@ -1888,3 +1888,51 @@ def test_symbols_uploads_data_pagination_bad_request(self):
 
         response = self.client.get(url, {'page': 'NaN'})
         eq_(response.status_code, 400)
+
+    def test_reprocessing(self):
+        url = reverse('manage:reprocessing')
+        response = self.client.get(url)
+        eq_(response.status_code, 302)
+
+        good_crash_id = '11cb72f5-eb28-41e1-a8e4-849982120611'
+        bad_crash_id = '00000000-0000-0000-0000-000000020611'
+
+        def mocked_post(crash_id):
+            if crash_id == good_crash_id:
+                return True
+            elif crash_id == bad_crash_id:
+                return
+            raise NotImplementedError(crash_id)
+
+        models.Reprocessing.implementation().post = mocked_post
+
+        self._login()
```

Comment:
There's not really any code that does it. We just have a django decorator. But it's not bad to have a little bit of basic testing to double-check that we really have that decorator on.